connectors/torus.ts

```diff
@@ -0,0 +1,20 @@
+// @ts-ignore
+const get = () => import(/* webpackChunkName: "torus" */ 'torus'); // v^2.0.6
```

Comment:
This package doesnt seem to be the correct one see https://www.npmjs.com/package/torus

Reply:
made the change!